src/handleSingleUser.ts

```diff
@@ -31,7 +32,9 @@ async function handleSingleUser(inputFields: InputFields, username: string, star
         prComments.push(repoData.prComments);
         prCommits.push(repoData.prCommits);
         prsCreated.push(repoData.prsCreated);
-    }));
+        sleep(2000);
+    });
```

Comment:
Thank you @smockle! I intended for there to be a 2000ms sleep between each Promise being creation, but maybe instead of map, I need to iterate and add a delay between each promise being created like:


```js
(async () => { 
  const names = ["cat", "dog", "fish"]; 

  const promises = [];
  for (const name of names) {
    promises.push(await request(name));
    await sleep(2000);
  
  }
  await Promise.all(promises); 
})();
```

Reply:
Marking this as resolved: We synced about this further on Zoom; subsequent commits have addressed the remaining issues!